lib/config/expression.cpp

```diff
@@ -553,8 +553,9 @@ ExpressionResult GetScopeExpression::DoEvaluate(ScriptFrame& frame, DebugHint *d
 		return frame.Self;
 	else if (m_ScopeSpec == ScopeGlobal)
 		return frame.GetGlobals();
-	else
-		VERIFY(!"Invalid scope.");
+
+	VERIFY(!"Invalid scope.");
+	return Empty;
```

Comment:
Which compiler did complain about this? I'm not able to find this kind of warning in the GHAs logs or locally on my machine. The previous version was pretty obvious that it will never return since it's negating a statically provided string, and I'm a bit surprised that the compiler isn't able to figure this out.